app/helpers/application_helper.rb

```diff
@@ -16,6 +16,10 @@ def yes_no_options
     ]
   end
 
+  def hide_nav_menu?
+    controller_name == 'sign_in_tokens' && (%w[validate sent].include? action_name)
+  end
+
```

Comment:
It feels icky for the superclass knowing about the subclass. An alternative could be defining

```
def hide_nav_menu?
  false
end
```

here

and overriding it withing the `SignInTokensController`?

Reply:
OK, moved the helper into the controller